sources/core/objects/Floor.cpp

```diff
@@ -36,7 +37,7 @@ void Floor::AttackBy(IdPtr<Item> item)
         if (!open_)
         {
             SetOpen(true);
-            Create<Item>(FloorTile::T_ITEM_S(), GetOwner());
+            IdPtr<FloorTile> ftile = Create<Item>(FloorTile::T_ITEM_S(), GetOwner());
```

Comment:
If `ftile` is not used then it is not needed to create it (just `Create<Item>(FloorTile::T_ITEM_S(), GetOwner())`)